<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stadium</class>
 <widget class="QMainWindow" name="Stadium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(255, 255, 255);
border-radius: 16px;
	border-color: rgb(0, 0, 127);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>372</width>
      <height>440</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
border-radius: 2px;
}</string>
       </property>
       <property name="text">
        <string>Stadium</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="ressources.qrc">:/FC resources/photo_2022-05-24_09-04-06.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Name </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Location </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Opening date </string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Reconstruction </string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Price </string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Architects </string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Capacity </string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Field size </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Estadio Santiago Bernab√©u </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Madrid, Spain </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> 14.12.1947 </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> 1982, 2001, 2021 </string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> 1,7 m. euro </string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Munoz Monasterio, Aleman Soler </string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> 81044 seats </string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> 105m x 68m </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_stadionsite">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 4px;
}
QPushButton:hover:!pressed { 
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>More about the stadium</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>445</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
border-radius: 2px;
}</string>
       </property>
       <property name="text">
        <string>Equipment and inverter</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_22">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: rgb(255, 255, 255);
	background-color: rgb(85, 170, 255);
border-radius: 2px;
}</string>
       </property>
       <property name="text">
        <string>Inverter cost calculator</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_23">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
	background-color: rgb(85, 170, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Number of chips </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_numfish">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 4px;
}</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_24">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
	background-color: rgb(85, 170, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Chip price </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_pricefish">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 4px;
}</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_25">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
	background-color: rgb(85, 170, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Number of bibs </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_numman">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 4px;
}</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_26">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
	background-color: rgb(85, 170, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Bib price </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_priceman">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
color: rgb(0, 0, 0);
	background-color: rgb(170, 255, 255);
border-radius: 4px;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_29">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color: rgb(85, 85, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string>Training room</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton_easy">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton{
color: rgb(255, 255, 255);
background-color: rgb(0, 255, 255);
border-radius: 4px;
}</string>
           </property>
           <property name="text">
            <string>Simplified</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_midl">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton{
color: rgb(255, 255, 255);
background-color: rgb(0, 255, 255);
border-radius: 4px;
}</string>
           </property>
           <property name="text">
            <string>Standard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_super">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton{
color: rgb(255, 255, 255);
background-color: rgb(0, 255, 255);
border-radius: 4px;
}</string>
           </property>
           <property name="text">
            <string>Advanced</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_27">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color: rgb(85, 170, 255);
border-radius: 2px;
}</string>
         </property>
         <property name="text">
          <string> Result </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_res">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color: rgb(0, 0, 0);
background-color: rgb(85, 255, 255);
border-radius: 4px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 4px;
}
QPushButton:hover:!pressed { 
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string> Back </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_resultt">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 4px;
}
QPushButton:hover:!pressed { 
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
